packages/head-tags/src/components/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { connect, Head } from "frontity";
+import { Connect } from "frontity/types";
+import HeadTagsPackage from "../../types";
+
+// Render all head tags from the current entity.
+const Root: React.FC<Connect<HeadTagsPackage>> = ({ state }) => {
+  return (
+    <Head>
+      {state.headTags.current.map(
```

Comment:
You can use `useMemo` for this computation (https://reactjs.org/docs/hooks-reference.html#usememo).